app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -5,7 +5,7 @@ module TextualSummary
     #
 
     def textual_group_properties
-      %i(name tag id)
+      %i(name tag id full_name)
```

Comment:
this is confusing since it is the registry's full name,  while the current file is the image's summary page